app/models/audit.rb

```diff
@@ -0,0 +1,99 @@
+require 'syslog'
+require 'logger'
+require 'time'
+
+module Audit
+  class << self
+    def notice msg, msgid, data
+      logger.info LogMessage.new msg, msgid, data, Syslog::LOG_NOTICE
```

Comment:
Where does the severity of `LOG_NOTICE` lie in terms of significance?

Reply:
Thing is, syslog and Ruby severity levels are different. Notice is between warn and info, eg. "Normal but significant conditions." Cf. https://en.wikipedia.org/wiki/Syslog#Severity_level

Reply:
Wouldn't it be great to have a comment in the code to this effect?